x-pack/plugins/beats_management/common/validations/interval.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import CronParser from 'cron-parser';
+import { FieldValue, FormData } from 'formsy-react';
+
+const isIntervalSuffix = (token: string): boolean => /([1-9][0-9]*[smhd])$/.test(token);
+
+const tokenize = (value: string): string[] =>
+  value.split(' ').reduce((acc: string[], cur: string) => {
```

Comment:
If your going to split on white space you should trim first or a harmless space at the end would fail your check

Reply:
Space at the end of the string is [ignored](https://github.com/elastic/kibana/pull/26219/files#diff-4b8ff706bf6eec79a13ae2bcf6d71b97R14) in the subsequent reduce call.

I've added a test to check that case though, didn't have it before.